operators/pkg/instctrl/cloudinit_test.go

```diff
@@ -116,21 +116,21 @@ var _ = Describe("Generation of the cloud-init configuration", func() {
 		}
 	})
 
-	ForgeWebDavSecret := func(usernameKey, passwordKey string) *corev1.Secret {
+	ForgePvcSecret := func(serviceNameKey, servicePathKey string) *corev1.Secret {
 		return &corev1.Secret{
-			ObjectMeta: forge.NamespacedNameToObjectMeta(webdavSecretName),
+			ObjectMeta: forge.NamespacedNameToObjectMeta(pvcSecretName),
 			Data: map[string][]byte{
-				usernameKey: []byte(webdavUsername),
-				passwordKey: []byte(webdavPassword),
+				serviceNameKey: []byte(NFSServiceName),
+				servicePathKey: []byte(NFSServicePath),
 			},
 		}
 	}
 
 	JustBeforeEach(func() {
 		client := FakeClientWrapped{Client: clientBuilder.Build()}
 		reconciler = instctrl.InstanceReconciler{
-			Client: client, Scheme: scheme.Scheme, WebdavSecretName: webdavCredentials,
-			ServiceUrls: instctrl.ServiceUrls{NextcloudBaseURL: nextcloudBaseURL},
+			Client: client, Scheme: scheme.Scheme,
+			ServiceUrls: instctrl.ServiceUrls{},
```

Comment:
ServiceUrls initialization could be dropped.

Reply:
Removed